api/routes/timeseries_forecast.go

```diff
@@ -114,7 +115,12 @@ func TimeseriesForecastHandler(metaCtor api.MetadataStorageCtor, dataCtor api.Da
 		}
 
 		// Recompute train/test split info for visualization purposes
-		split := compute.SplitTimeSeries(timeseries.Timeseries, compute.TimeSeriesTrainTestSplitThreshold)
+		config, err := env.LoadConfig()
```

Comment:
Instead of loading the config in the route, pass it in as a parameter (either the whole config struct or the split threshold). The cluster route handler has as parameter the whole config struct. Quite a few routes have specific config parameters.

Reply:
Done